recipes/pyabel/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "PyAbel" %}
+{% set version = "0.7.4" %}
+{% set sha256 = "95f40daeba34789119aa2391a113248763f333e567b9d65fbd3a0fde1b10ed1f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - toolchain
+    - cython
+    - numpy >=1.6
```

Comment:
What does `- numpy x.x` do?

@rth knows the Cython side of PyAbel the best, so he may have input. But yes, we have `cimport numpy as np`, so I think that means that we have compiled extensions that link to numpy.